apps/server/src/v1/statusReportUpdates/post.ts

```diff
@@ -81,31 +80,24 @@ export function registerPostStatusReportUpdate(
 
     // send email
 
-    if (workspacePlan.title !== "Hobby") {
-      const allPages = await db
+    if (workspacePlan.title !== "Hobby" && _statusReport.pageId) {
```

Comment:
Lets use the dynamic getLimit function.

```suggestion
    if (getLimit(workspacePlan, "status-subscribers") && _statusReport.pageId) {
```